erpnext/stock/doctype/stock_entry/stock_entry.json

```diff
@@ -638,12 +639,18 @@
    "label": "Harvest",
    "options": "Harvest",
    "read_only": 1
+  },
+  {
+   "fetch_from": "work_order.purchase_receipt",
+   "fieldname": "purchase_receipt",
+   "fieldtype": "Data",
```

Comment:
@meshramsaroj  Field type should be read-only

Reply:
made field type read only